docs/en/contributing.rst

```diff
@@ -130,17 +157,17 @@ For example, in the sketch:
 .. code-block:: arduino
 
     /*
-      THIS FEATURE IS SUPPORTED ONLY BY ESP32-S2 AND ESP32-C3
+      THIS FEATURE REQUIRES WI-FI SUPPORT AND IS NOT AVAILABLE FOR ESP32-S2 AS IT DOES NOT HAVE ENOUGH RAM.
     */
 
 And in the ``README.md`` file:
 
 .. code-block:: markdown
 
-    Currently, this example supports the following targets.
+    Currently, this example requires Wi-Fi and supports the following targets.
 
-    | Supported Targets | ESP32 | ESP32-S2 | ESP32-C3 | ESP32-S3 |
-    | ----------------- | ----- | -------- | -------- | -------- |
+    | Supported Targets | ESP32 | ESP32-H2 | ESP32-S3 | ESP32-C3 | ESP32-C6 |
+    | ----------------- | ----- | -------- | -------- | -------- | -------- |
```

Comment:
Just a comment here. I know its an example of the configuration, but H2 does not have wifi so idk if its good example here :D

Reply:
Ooops, missed that. Thanks!